install_config/topics/escaping_special_characters.adoc

```diff
@@ -0,0 +1,17 @@
+////
+Escaping special characters in ansible variables file
+
+This module included in the following assemblies:
+* install_config/configuring_openstack.adoc
+* install_config/install/advanced_install.adoc
+////
+
+[IMPORTANT]
+====
+If you are using special characters (for example, `pass:[#]`, `{` or `}`) in the
```

Comment:
The syntax for an ini inventory file is ```pass=whatever``` instead ```pass:whatever```
I'd also use ```pass=mycomplexpa##word``` instead ```pass=[#]```

Reply:
My bad, the `pass:[...]` format is the asciidoc passthrough variable to render the text correctly.

The final rendered text will looks like this:
> If you are using special characters (for example, `#`, `{` or `}`) in the
> Ansible host inventory file, you need to double-escape them to make it work. For
> example, to use `mypasswordwith###hashsigns` as a value for the variable
> `openshift_cloudprovider_openstack_password`, declare it as
> `openshift_cloudprovider_openstack_password='"mypasswordwith###hashsigns"'` in
> the Ansible host inventory file.